app/src/main/java/eu/kanade/tachiyomi/data/backup/full/models/BackupTracking.kt

```diff
@@ -25,11 +25,17 @@ data class BackupTracking(
     @ProtoNumber(10) var startedReadingDate: Long = 0,
     // finishedReadingDate is called endReadTime in 1.x
     @ProtoNumber(11) var finishedReadingDate: Long = 0,
+    @ProtoNumber(12) var mediaId: Long = 0,
```

Comment:
Since its a Tachiyomi 0.x specific value it should start at 100
```suggestion
    @ProtoNumber(100) var mediaId: Long = 0,
```